README.md

```diff
@@ -23,15 +17,25 @@ git clone https://github.com/hackersatcambridge/hac-website.git
 And then run:
 
 ```
-npm install
+docker-compose up
 ```
 
-followed by:
+This will build all the files and run the web server with appropriate output. It will also watch for changes to the files and rebuild as necessarily. You can stop this process at any time with `^C` (`ctrl` + `C`).
+
+### What `docker-compose up` is doing
+
+This command starts up a docker container that has all the required dependencies, then starts up a development server.
+
+Internally, this command runs the Swift and static file (CSS, images, etc.) build processes in parallel. When they complete, it runs the executable produced by the Swift build. It exposes a server on http://localhost:3000. For convenience, it will reload the browser window for you whenever you make a relevant change to the files. This is powered by BrowserSync, and you can access the interface for it at http://localhost:3001.
```

Comment:
Good one, I'll add a note. I think it's not necessary to point to Dockerfile/gulpfile as that's standard and can be discovered by reading the docs for the relevant technologies.